torch/fx/graph.py

```diff
@@ -544,6 +545,11 @@ def emit_node(node : Node):
                 elif isinstance(meta_val, TensorMetadata):
                     maybe_type_annotation = f': "{dtype_abbrs[meta_val.dtype]}{stringify_shape(meta_val.shape)}"'
 
+                # Add a layout type annotation for sparse tensors
+                # (purely to distinguish them from dense tensors).
+                if tensor_meta is not None and tensor_meta.layout is not None:
+                   maybe_type_annotation = f'{maybe_type_annotation}:{tensor_meta.layout}'
```

Comment:
This can be simplified to
```suggestion
                    maybe_type_annotation = f': "{dtype_abbrs[meta_val.dtype]}{stringify_shape(meta_val.shape)}":{tensor_val.layout}'
```